script.js

```diff
@@ -0,0 +1,155 @@
+//DOM요소 가져오기
+const dateInfo = document.getElementById('date');
+const input = document.querySelector('input');
+const form = document.querySelector('form');
+const todoContent = document.querySelector('.todoContent');
+const doneContent = document.querySelector('.doneContent');
+const todoNum = document.getElementById('todoNum');
+const doneNum = document.getElementById('doneNum');
+
+let todoList = [];
+let doneList = [];
+
+//날짜 띄우는 함수
+const clock = () => {
+  let current = new Date(); //현재 날짜 와 시간
+  let year = current.getFullYear(); //연도
+  let month = current.getMonth(); //월
```

Comment:
아래에서 month +1을 너무 잘 해주셨지만, 애초에 선언할 때
```suggestion
  let month = current.getMonth() + 1; //월
```
과 같이 선언해주시면 아래에서 재사용을 할 일이 있을 때 바로 사용하기 좋을 것 같습니다.